stdlib/2and3/imghdr.pyi

```diff
@@ -1,16 +1,21 @@
-from typing import overload, Union, Text, BinaryIO, Optional, Any, List, Callable
-import sys
 import os
+import sys
+from typing import Any, BinaryIO, Callable, List, Optional, Protocol, Text, Union, overload
+
+class _ReadableBinary(Protocol):
+    def tell(self) -> int: ...
+    def read(self, size: int = ...) -> bytes: ...
```

Comment:
Minor: You could leave out the requirement to have a default value, since it's not required by `what()`.

Reply:
Done.